+++ tensorflow-1.0.0-rc2/third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc.tpl        2017-02-15 20:40:47.633496842 +0100
--- tensorflow-1.0.0-rc2/third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc.tpl        2017-02-08 19:32:14.000000000 +0100
@@ -46,13 +46,13 @@ import sys
 import pipes
 
 # Template values set by cuda_autoconf.
-CPU_COMPILER = ('%{cpu_compiler}')
-GCC_HOST_COMPILER_PATH = ('%{gcc_host_compiler_path}')
+CPU_COMPILER = os.environ['SPACK_CC']
+GCC_HOST_COMPILER_PATH = CPU_COMPILER
 
 CURRENT_DIR = os.path.dirname(sys.argv[0])
 NVCC_PATH = CURRENT_DIR + '/../../../cuda/bin/nvcc'
-LLVM_HOST_COMPILER_PATH = ('/usr/bin/gcc')
-PREFIX_DIR = os.path.dirname(GCC_HOST_COMPILER_PATH)
+LLVM_HOST_COMPILER_PATH = os.environ['SPACK_CC']
+PREFIX_DIR = os.path.dirname(os.environ['SPACK_CC'])
 NVCC_VERSION = '%{cuda_version}'
 
 def Log(s):

